recipes/numpy/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set version="1.11.0" %}
+
+package:
+  name: numpy
+  version: {{ version }}
+
+source:
+  fn: numpy-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/n/numpy/numpy-{{ version }}.tar.gz
+  md5: bc56fb9fc2895aa4961802ffbdb31d0b
+  patches:
+    - osx-dist.patch          # [osx]
```

Comment:
I'm asking because I haven't needed this patch building NumPy before so am a bit suspicious as to whether it is actually necessary.